.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+node_js:
+    - "4.2"
+before_install:
+    - npm install -g grunt-cli
+    - npm install -g bower
+    - bower install
+    - npm update
```

Comment:
if you run tests on travis, you might run into trouble because of grunt and grunt plugins. If you update npm to version3, version2 gets installed by default, then it will warn instead of fail. `npm install -g npm` will accomplish that if you ever need that.